code/style.css

```diff
@@ -1,9 +1,156 @@
-/* 
-If you can see a peach background, the css file is correctly linked. 
-Feel free to change or remove this styling! 
-We want you focus A LOT on the styling for this project. 
-*/
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+
+* {
+    box-sizing: border-box;
+}
+
+form {
+    display: table;
+    max-width: 48em;
+    margin: auto;
+    text-align: center;
+}
 
 body {
-  background-color: peachpuff;
+    font-size: 16px;
+    font-family: 'Montserrat', sans-serif;
+    background: #f8fbfd;
+    color: #525f7f;
+    padding: 40px;
+}
+
+h2 {
+    text-transform: uppercase;
+    font-size: 16px;
+    margin-top: 0;
+    text-align: left;
+}
+
+fieldset {
+    border: none;
+    background: white;
+    border-radius: 4px;
+    box-shadow: 0 1px 3px 0 rgba(50, 50, 93, 0.15), 0 4px 6px 0 
+    rgba(112, 157, 199, 0.15);
+}
+
+label {
+    display: flex;
+    border-bottom: 1px solid #f0f5fa;
+    align-items: center;
+    height: 50px;
+}
+
+label:last-child {
+    border-bottom: none;
+}
+
+span {
+    width: 30%;
+    text-align: right;
+    margin-right: 15px;
+    flex-shrink: 0;
+}
+
+input {
+    width: 100%;
+    font-size: 15px;
+    padding: 10px 0;
+    appearance: none;
+    border: none;
+    outline: none;
+}
+
+.input:after {
+    display: table;
+    clear: both;
 }
+
+select {
+    width: 100%;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    margin-right: 10px;
+    appearance: none;
+    -webkit-appearance: none;
+    background-color: transparent;
+    border: none;
+    font-size: 14px;
+    color: #313b3f;
+    outline: none;
+}
+
+::-webkit-input-placeholder {
+    color:#aab7c4;
+ }
+ 
+ :-moz-placeholder { /* Firefox 18- */
+    color: #aab7c4;  
+ }
+ 
+ ::-moz-placeholder {  /* Firefox 19+ */
+    color: #aab7c4;  
+ }
+ 
+ :-ms-input-placeholder {  
+    color: #aab7c4;  
+ }
+
+ /*---checkbox---*/
+
+ input[type="checkbox"] {
+    display:none;
+  }
+
+  input[type="checkbox"] + label::before {
+    width: 15px;
+    height: 15px;
+    border-radius: 15px;
```

Comment:
It's a bit confusing that this checkbox looks like a radio button 🙈 Consider getting rid of the `border-radius` for better UI